src/python/ViewerBinding.cpp

```diff
@@ -2,8 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "TypeCasters.h"
```

Comment:
I know this is already merged but this line needs to stay there to resolve #168.